utilities/class.tapestry-helpers.php

```diff
@@ -307,4 +307,125 @@ public static function nodeNeighbourIsPublished($nodeMetaId, $tapestryPostId)
         }
         return false;
     }
+
+    /**
+     * Update the Kaltura upload status of a video node.
+     * 
+     * @param TapestryNode      $node           Video node to update
+     * @param string            $newStatus      Upload status
+     * @param MediaEntry|null   $kalturaData    (optional) Response from Kaltura API
+     */
+    public static function saveVideoUploadStatusInNode($node, $newStatus, $kalturaData = null)
+    {
+        $typeData = $node->getTypeData();
+        $typeData->kalturaData['uploadStatus'] = $newStatus;
+
+        if ($kalturaData) {
+            $typeData->kalturaData['id'] = $kalturaData->id;
+            $typeData->kalturaData['partnerId'] = $kalturaData->partnerId;
```

Comment:
Is this particular `partnerId` property being used? I did a search and couldn't find any other occurrences.

Reply:
Yes, I think it is not used anywhere. It was saved in the original upload code, though I am not sure why.

Since it's not being used, I will just delete it for now and it should be easy to add back in the future.